ERCS/erc-3643.md

```diff
@@ -1,10 +1,11 @@
 ---
-eip: 3643
+erc: 3643
 title: T-REX - Token for Regulated EXchanges
 description: An institutional grade security token contract that provides interfaces for the management and compliant transfer of security tokens.
 author: Joachim Lebrun (@Joachim-Lebrun), Tony Malghem (@TonyMalghem), Kevin Thizy (@Nakasar), Luc Falempin (@lfalempin), Adam Boudjemaa (@Aboudjem)
 discussions-to: https://ethereum-magicians.org/t/eip-3643-proposition-of-the-t-rex-token-standard-for-securities/6844
-status: Review
+status: Last Call
+last-call-deadline: 2023-11-01
```

Comment:
```suggestion
last-call-deadline: 2023-11-20
```

If you give maintainers the permission to edit your branch, we can update your last call deadline just before merging.

Reply:
Hey! It seems we can't have this option because the fork is under a GitHub organization (@erc-3643) and not a personal account. We can add you as editor on the fork repo though.